shared/storage/minio.py

```diff
@@ -17,13 +17,29 @@
 )
 from minio.deleteobjects import DeleteObject
 from minio.error import MinioException, S3Error
+from urllib3.response import HTTPResponse
 
-from shared.storage.base import CHUNK_SIZE, BaseStorageService
```

Comment:
you could keep the `CHUNK_SIZE` constant instead of hardcoding 16k in all the places.